src/cljs/main/broadfcui/page/library/library_page.cljs

```diff
@@ -213,7 +213,7 @@
   [:div {:style {:textOverflow "ellipsis" :overflow "hidden"}}
    (if-not (string/blank? highlight)
      (let [suggestion-parts (string/split suggestion highlight)]
-       [:span {} (first suggestion-parts) [:strong {} highlight] (last suggestion-parts)])
+       [:span {} (first suggestion-parts) [:strong {} highlight] (second suggestion-parts)])
```

Comment:
Since it seems like we can have multiple matches:
```clojure
[:span {} (interpose [:strong {} highlight] (string/split suggestion highlight))]
```

Reply:
the `interpose` approach only works if the `split` returns multiple items, so now I have some conditional logic in place to look at the number of items ...